src/app/components/users/users.component.ts

```diff
@@ -52,31 +53,31 @@ export class UsersComponent implements OnInit {
   deleteUser(username: string) {
     const dialogRef = this.dialog.open(DeleteDialogComponent);
     dialogRef.afterClosed().subscribe(data => {
-      if (data == "true") {
+      if (data == 'true') {
         this.service.deleteUser(username).subscribe(() => {
           this.getAllUsers();
         });
       }
-    })
+    });
   }
 
   getAllUsers() {
     let clients = this.service.getAllUsers();
     clients.subscribe(data => {
```

Comment:
scrie in acelasi fel, pentru ce e declarat variabila clients?

Reply:
Extra code, clients nu face nimic acolo.